.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,39 @@
+name: Build and test
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install apt dependencies
+      run: |
+        sudo add-apt-repository -y ppa:nextgis/ppa
+        sudo apt update
+        sudo apt-get install libgdal-dev
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install Cython
+        pip install GDAL==$(gdal-config --version) --global-option=build_ext --global-option="-I/usr/include/gdal -L/usr/local/lib -lgdal"
+        pip install .[tests]
+    - name: Lint with flake8
+      run: |
+        pip install flake8
+        # stop the build if there are Python syntax errors or undefined names
+        # flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
```

Comment:
Does the build fail, when `flake8` returns with a non-zero exit code? If yes, is there an "allow failure" option?

Reply:
It does. Unfortunately, it does not give a visual indication when continue-on-error is enabled and the test failed. It shows the same green checkmark as for a successful test.

Reply:
Oh, but it does show an annotation indication the failure. So that's good.